index.html

```diff
@@ -3857,6 +3857,72 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section class="informative">
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+Prior to verification when using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation, as described in Section
+<a href="#data-schemas"></a>.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into databases that index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed for securing mechanisms that require an understanding of
+and processing of JSON-LD.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
+is desired, an implementer is advised to follow the following rule:
+          </p>
+
+          <ul>
+            <li>
+Ensure that all values associated with a `@context` property are in the
+expected order, the contents of the context files match known good
+cryptographic hashes for each file, and the contents are known to be valid.
+            </li>
+          </ul>
+
+          <p>
+Using static context files with a JSON Schema is one acceptable approach to
+implementing the rule above. This can ensure proper term identification,
+typing, and order, when a JSON document is processed as JSON-LD.
+          </p>
+
+          <p>
+The rule above guarantees semantic interoperability between JSON and JSON-LD for
+literal JSON keys mapped to URIs by the `@context` mechanism. While JSON-LD processors will use the
+specific mechanism provided and can verify that all terms are correctly
+specified, JSON-based processors implicitly accept the same set of terms without
+testing that they are correct. In other words, the context in which the data
+exchange happens is explicitly stated for both JSON and JSON-LD by using the
+same mechanism. With respect to JSON-based processors, this is achieved in a
+lightweight manner, without having to use JSON-LD processing libraries.
```

Comment:
What does semantic interoperability mean in terms of concrete example? IMO, this term is quite overloaded. IMO, there is no semantic interoperability if JSON-only and JSON-LD implementers if they use the default `@vocab` only.

Reply:
Yes, good point. It might be best to not use the word and just describe what is achieved.

"semantic interoperability" means that the terms have one meaning whether they are interpreted as JSON-LD or if they're interpreted as JSON (because they use the same mechanism for how the terms are defined). An analogy is using a single source file and executing it via runtime interpretation vs. JIT compiling and running the compiled form. The output of the program in both cases is exactly the same, but how each system interprets the semantics of the source file (runtime interpretation vs. compiled executable) is different. It's the output of the program that matters, and that's entirely dependent on the semantics expressed by the source file being the same between both execution environments. IOW, semantic interoperability is proven when the output of the program, for the same inputs, is indistinguishable between the runtime vs. compiled execution of the program.

I'd argue that there is semantic interoperability between  JSON-only and JSON-LD implementers if they use the default @vocab only... both interpretations will lead to the same semantics... that undefined terms are issuer defined. So, I believe the "semantic interop" statement holds given the definition above... but I'll try re-writing the section to avoid the use of the term given that it has to be explained.